CrewChiefV4/RF2/RF2GameStateMapper.cs

```diff
@@ -484,10 +485,11 @@ public GameStateData mapToGameStateData(Object memoryMappedFileStruct, GameState
             // If any difference between current and previous states suggests it is a new session
             if (pgs == null
                 || csd.SessionType != psd.SessionType
-                || !string.Equals(cgs.carClass.getClassIdentifier(), pgs.carClass.getClassIdentifier())
+                // TODO: make sure we don't need this.
+                /*|| !string.Equals(cgs.carClass.getClassIdentifier(), pgs.carClass.getClassIdentifier())
```

Comment:
tricky... we probably don't need it - if detecting a car class change is the only way we can detect a new session we have big problems. Happy to remove this, but worth leaving the comment in there